### ISSUE TITLE
What is 'D_canvas'？

### ISSUE THREAD

**Issue by 123dddwn** [afruehstueck/insetGAN#13]
Hi! May I ask you two questions about G_canvas and D_canvas?
1. In the code I found G_canvas is defined but D_canvas is not defined. So how to solve this problem?
2. In the paper, I didn't read anything about discriminator in the paper. But why something about discriminator appears in the code? 
Thank you.

**Comment by afruehstueck** [comment]
One of the many losses we experimented with was to add a discriminator loss to the optimization to improve the quality of the full body. Since we did not find this to contribute to our results, we did not use the discriminator loss in either our final algorithm or the paper, but it may still appear in the code as one of the options we explored. You can disregard it, delete the method `def disc_loss(optim)` or load the trained discriminator to `D_canvas` to experiment with, if you would like to do so.

**Comment by 123dddwn** [comment]
ckpt = f'{config.home_dir}/networks/DeepFashion_1024x768.pkl', I didn't find this path. What should I do to solve this probelm?

**Comment by afruehstueck** [comment]
Please download the checkpoint from [here](https://github.com/afruehstueck/insetGAN#pre-trained-models) according to the instructions and put it in the appropriate directory.

**Comment by 123dddwn** [comment]
Thank you! 😊
